### PR TITLE
changes to support CC use case

### PR DESCRIPTION
**Pull request by rafsalas19**

### CODE REVIEW

File: run-health-checks.sh [hunk 1]
```diff
@@ -35,11 +39,8 @@ elif  echo "$SKU" | grep -q "hx176-48rs"; then
 elif  echo "$SKU" | grep -q "hx176-24rs"; then
     conf_name="hx176-24rs"
 else
-    echo "SKU health check currently not implemented"
-    exit 1
+    echo "SKU is currently not supported by Azure health checks." | tee -a $log_path
```
Comment (bwatrous) [inline run-health-checks.sh]: Probably worth adding the sku name to the message as long as we're touching the line.  "SKU $SKU..."

Comment (rafsalas19) [reply run-health-checks.sh]: Let me address

Comment (rafsalas19) [reply run-health-checks.sh]: addressed

**Comment by rafsalas19** [comment]
I made a small change to ib_write_gdr test to remove a log message that wasn't useful.

**Review by jithinjosepkl — APPROVED** [review]

**Review by AnastasiiaEvdokimova — APPROVED** [review]

**Pull request merged**